Modules/ConfigOptions.lua

```diff
@@ -156,6 +156,14 @@ return {
 	{ var = "perforateSpikeOverlap", type = "count", label = "# of Overlapping Spikes:", tooltip = "Affects the DPS of Perforate in Blood Stance.\nMaximum is limited by the number of Spikes of Perforate.", ifSkill = "Perforate", apply = function(val, modList, enemyModList)
 		modList:NewMod("Multiplier:PerforateSpikeOverlap", "BASE", val, "Config", { type = "SkillName", skillName = "Perforate" })
 	end },
+	{ label = "Pride:", ifSkill = "Pride" },
+	{ var = "prideEffect", type = "list", label = "Pride Aura Effect:", list = {{val="MIN",label="Initial effect"},{val="MAX",label="Maximum effect"}}, apply = function(val, modList, enemyModList)
```

Comment:
The formatting on this line is inconsistent. There should be whitespace around all curly brackets.

Reply:
None of the other skills with dropdowns like this are formatted to have spaces around the curly brackets